### DISCUSSION TITLE
Custom Project setup

### DISCUSSION THREAD

**Discussion by mwakazi-dev** [DevSkillsHQ/discusssions#76] [Help] [answered]
In this case I am not limited to how I setup my project. I can create the project the way I want it, is that true?

**Comment by fenske** [comment, marked as answer]
Hi @jmwakz99.

> I can create the project the way I want it, is that true?

Ofc. Just beware that there is some custom setup needed to make the autotests work (see the README).

Best of luck!

BR,
Anton.

**Reply by mwakazi-dev** [reply]
Sure. Thanks.
